File: week-09/day-01/34.js
'use strict';

var ag = [3, 4, 5, 6, 7];
var i = 0
// double all the element's values in ag

while (i < ag.length) {
   ag[i] *= 2;
   i++;
}
console.log(ag)

for (var i = 0; i < ag.length; i++) {
  ag[i] *= 2;
}
console.log(ag);

File: week-03/day-01/34.py
ag = [3, 4, 5, 6, 7]

#while
i = 0
b = len(ag)
while i < b :
    ag[i] *= 2
    i += 1
print(ag)

# for
for i in range(len(ag)):
    ag[i] *= 2
print(i)

### element double ###
ag = [3, 4, 5, 6, 7]

#while
i = 0
b = len(ag)

while i < b :
    print()
    ag += [ag[i]]
    i += 1
print(ag)

# for
ag = [3, 4, 5, 6, 7]
i = 0
b = len(ag)

for i in range(b) :
    ag += [ag[i]]
print(ag)
